code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -430,7 +430,7 @@
 	max_ammo_count = 1
 	ammo_name = "area denial sentry"
 	travelling_time = 0 // handled by droppod
-	point_cost = 800
+	point_cost = 600
```

Comment:
```suggestion
	point_cost = 700
```
Assuming you have about ~6k points left after buying dropship weapons, equipment, and ammo (starting w/ 10k), this will allow you to buy 8 sentries as opposed to the previous 7. At a regen rate of 18 points/min, one sentry will take 39 minutes to be earned as opposed to 45.